packages/utils/regl-renderer/src/rendering/render.js

```diff
@@ -56,8 +56,8 @@ export const prepareRender = (params) => {
                 drawCmd = drawCache2.get(visuals.cacheId)
               } else {
                 visuals.cacheId = drawCache2.size
-console.log(visuals.drawCmd)
-console.log(context)
+                console.log(visuals.drawCmd)
```

Comment:
let's remove the debug

Reply:
I removed these console.log statements, as well as a bunch of other ones. Many more remain, for another day.